mecanumbot_lidar_detect_people:
  ros__parameters:
    weight_file: dr_spaam_5_on_frog.pth
    conf_thresh: 0.3
    stride: 1
    scan_topic: scan
    detections_topic: dr_spaam/dets
    rviz_topic: dr_spaam/marker